shell/platform/fuchsia/dart_runner/embedder/BUILD.gn

```diff
@@ -2,38 +2,65 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import("//build/dart/toolchain.gni")
-import("//topaz/runtime/dart/dart_component.gni")
-import("//topaz/runtime/dart/dart_kernel.gni")
+import("//build/compiled_action.gni")
+import("//third_party/dart/build/dart/dart_action.gni")
+import("$flutter_root/common/fuchsia_config.gni")
+import("$flutter_root/tools/fuchsia/dart.gni")
 
-dart_kernel("shim") {
-  platform_name = "dart_runner"
-  platform_deps = [ "//topaz/runtime/dart_runner/kernel:kernel_platform_files" ]
-  platform_path = "$root_out_dir/dart_runner_patched_sdk"
+template("dart_shim_kernel") {
```

Comment:
note that per my conversation with @rmacnak-google since we only go through this in-order to setup an isolate. Product mode here should not really matter.